test/scala/co/actioniq/luna/DAOSpec.scala

```diff
@@ -129,6 +129,36 @@ class DAOSpec extends Specification with Mockito {
       newPlayer.name mustEqual "Mary"
       newPlayer.teamId mustEqual 1L
     }
+    "update two field" in new TestScope with NoopLoggerProvider {
+      val player = awaitResult(playerDao.readByIdFuture(larryId)).get
+      val updateFuture = playerDao.updateFieldFuture[String, Long](
+        larryId,
+        row => (row.name, row.teamId),
```

Comment:
Having more than one seems dangerous as it assumes that you're putting it in the right order. Would it be possible to use a map instead? `(row.name -> "boooo", row.teamId -> 2L)`?

Reply:
not sure how possible typing this would be though